code/modules/spells/spell_code.dm

```diff
@@ -222,11 +222,6 @@ var/list/spells = typesof(/spell) //needed for the badmin verb for now
 		to_chat(user, "<span class='warning'>You shouldn't have this spell! Something's wrong.</span>")
 		return 0
 
-	var/spell_leech = user.disrupts_psionics()
```

Comment:
maybe do 'disrupts_spells()' with dunno nullrod by default, and override with psi stuff in pack?